src/intro/no-std.md

```diff
@@ -23,12 +23,31 @@ This means the rust [stdlib](https://doc.rust-lang.org/std/) and dynamic memory
 However, such code can use [libcore](https://doc.rust-lang.org/core/), which can easily be made available
 in any kind of environment by providing just a few symbols (for details see [libcore](https://doc.rust-lang.org/core/)).
 
+### The stdlib Runtime
+As mentioned before using stdlib requires some sort of system integration, but this is not only because
```

Comment:
Small nitpick, you might want to use `libstd` across this page rather than `stdlib`. I think the former is more common in Rust nomenclature

Reply:
good point, thx -> I'll update the PR